### ISSUE TITLE
The search bar is too down

### ISSUE THREAD

**Issue by Saran-pariyar** [jenkoian/hacktoberfest-checker#719]
When we enter the website, we cannot directly see the search bar. I can make UX better by making the search bar appear on the screen directly when we enter the website without scrolling.
#### As you see we need to scroll to see the search bar
Please assign this to me and I will make the styling changes 

![image](https://user-images.githubusercontent.com/86418083/194733456-250f8d7b-f736-4850-9ee3-78e7e44c50b6.png)

**Comment by gitamandeep** [comment]
Sir, please assign this task to me. I can work on it.

**Comment by jenkoian** [comment]
Yeah good point, I guess the split heading and size causes this. Yeah go for it, don't worry about being assigned.
